libcst/_nodes/tests/test_with.py

```diff
@@ -178,6 +180,18 @@ class WithTest(CSTNodeTest):
                 "parser": parse_statement,
                 "expected_position": CodeRange((1, 0), (1, 36)),
             },
+            # Parenthesized
```

Comment:
Don't forget to add a test case with multiple `WithItem`s spanning multiple lines